css/order.css

```diff
@@ -340,4 +340,73 @@ color:inherit;
   100% {
     transform: rotate(360deg);
   }
+}
+
+.rotateimgbtc {
+  -webkit-transform:rotate(15deg);
+  -moz-transform: rotate(15deg);
+  -ms-transform: rotate(15deg);
+  -o-transform: rotate(15deg);
+  transform: rotate(15deg);
+  float: left;
+  margin-right: 10px;
+  padding-left: 5px;
+}
+
+.rotateimgbch {
+  -webkit-transform:rotate(-15deg);
+  -moz-transform: rotate(-15deg);
+  -ms-transform: rotate(-15deg);
+  -o-transform: rotate(-15deg);
+  transform: rotate(-15deg);
+  float: left;
+  margin-right: 10px;
+  padding-left: 5px;
+} 
+
+.bnomics-select-container {
+  text-align:center;
+  max-width:450px;
+  margin:auto;
+}
+
+.bnomics-select-options {
+  cursor:pointer;
+}
+
+.bnomics-select-options h3 {
+  display:inline;
+  margin: 0;
+}
+
+.bnomics-select-container table{
+  border-spacing: 0 1em;
```

Comment:
Some themes still showing the border around the table. Lets use `border: 0;` to remove the border.